README.md

```diff
@@ -0,0 +1,62 @@
+# Betterbird (Flathub)
+
+Betterbird is a fine-tuned version of [Mozilla Thunderbird](https://www.thunderbird.net/), Thunderbird on steroids, if you will.
+
+[Betterbird](https://betterbird.eu/) for [Flatpak](https://flatpak.org/) installation instructions are available by [clicking here to visit the Betterbird app page on Flathub](https://flathub.org/apps/details/eu.betterbird.Betterbird).
+
+
+## Migration from pre-exisiting installations
```

Comment:
just so you are aware, no user will see this, It is not displayed anywhere but here.

Reply:
Ok, thanks for pointing that out.